docs/customizations/parts/code-samples-example.md

```
@@ -0,0 +1,68 @@
+<script setup lang="ts">
```

Markdownlint

Element: script
Inline HTML

(MD033, no-inline-html)